src/app/redux/selectors/custom-card.selector.ts

```diff
@@ -0,0 +1,12 @@
+import { createSelector } from '@ngrx/store';
+import { CustomCardState } from '../state/models/custom-card.state';
+import { YoutubeClientState } from '../state/youtube-client.state';
+
+// tslint:disable-next-line: typedef
```

Comment:
Why do you decide to disable tslint rule instead of define type? This is bad practice